content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md

```diff
@@ -46,90 +46,91 @@ Enter the name you want to use for the administrator account.
 
 Enter and confirm the admin user passwords.
 
-Select the **root**, **builtin_administrators** and **builtin_users** groups from the **Auxiliary Group** dropdown list.
+Select the **builtin_administrators** group from the **Auxiliary Group** dropdown list.
 
 {{< trueimage src="/images/SCALE/22.12/AddingAdminUserAuxiliaryGroup.png" alt="Add Admin User to builtin_administrators" id="2: Add Admin User to builtin_administrators" >}}
 
-Click **Save**.
-
-Log out of the TrueNAS system and then log back in using the admin user credentials. Once you are back in the TrueNAS web UI, determine that the admin user credentials are working properly with your network configuration.
-
-### Configuring the Admin User Account for Nextcloud App
-The Nextcloud application, configured on the **Apps > Available Applications** screen, requires including sudo permissions for the administrator account. 
-To verify, or manage the local administrator account, go to **Credentials > Local Users** and click on the administrator user row to expand it, then click **Edit** to open the **Edit User** configuration screen.
-
 Scroll down to the **Authentication** settings and select the proper **Allow sudo** authorization settings.
+Some applications, such as [Nextcloud]({{< relref "InstallNextCloudMedia.md" >}}), require sudo permissions for the administrator account.
+By default, TrueNAS SCALE sets admin account authorization to **Allow all sudo commands**.
+Click the checkbox to choose this configuration.
 
 Click **Save**.
 
-For information on adding sudo permissions in cli commands, see [Installing Nextcloud for Media Previews]({{< relref "InstallNextCloudMedia.md" >}}).
+Log out of the TrueNAS system and then log back in using the admin user credentials. Once you are back in the TrueNAS web UI, determine that the admin user credentials are working properly with your network configuration.
 
 ## Creating User Accounts
 
 To create a new user, click **Add**.
 
-TrueNAS lets users configure four different user account traits (settings). 
-
 ### Configuring User Identification Settings
 
-{{< trueimage src="/images/SCALE/22.12/AddUserIdentificationSettings.png" alt="Add User Identification Settings" id="3: Add User Identification Settings" >}}
+{{< trueimage src="/images/SCALE/23.10/AddUserIdentificationSettings.png" alt="Add User Identification Settings" id="3: Add User Identification Settings" >}}
 
-Enter the user full name in **Full Name**.
-TrueNAS suggests a simplified name in **Username** derived from the **Full Name**, but you can override it with your own choice.
+Enter the user's full name in **Full Name**.
```

Comment:
do not use the possessive case
```suggestion
Enter the user full name in **Full Name**.
```

Reply:
Ok. I find the original unclear, so I'm working on options with Tim